x/service/types/params.go

```diff
@@ -29,6 +35,14 @@ func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 
 // Validate validates the set of params
 func (p Params) Validate() error {
+	// TODO(@h5law): Look into better validation
+	if p.AddServiceFee < 1000000 { // 1 POKT
```

Comment:
@Olshansk I like the idea of validating against the service fee default value - still unsure on adding the `Minimum` could you explain a bit more around your reasoning here?

Reply:
Yea, I was thinking that it would be like variable tx fees (at the time of writing this comment), but believe it overcomplicated things so let's ignore.